tests/ha/test_ha_pod_restart.py

```diff
@@ -2960,12 +2961,21 @@ def test_continuous_reads_during_pod_restart_ros(self):
         LOGGER.info("Step 8: Cluster is in good state. All the services are up and running")
         event.clear()
         thread.join()
-
+        LOGGER.debug("Event is cleared and thread has joined.")
         LOGGER.info("Verifying responses from background process")
         responses = {}
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not responses:
+            assert_utils.assert_true(False, "Background S3bench Failures")
+        LOGGER.debug("Background S3bench responses : %s", responses)
+        if not responses["pass_res"]:
+            assert_utils.assert_true(False,
+                                     "No background IOs passed while event was cleared")
         nonbkgrd_logs = list(x[1] for x in responses["pass_res"])
+        if not responses["fail_res"]:
+            assert_utils.assert_true(False,
+                                     "No background IOs passed while event was set")
```

Comment:
> No background IOs failed*

Changing it to 'response', instead of pass fail, as responses["fail_res"] will contain IOs result and our check if to verify that only.